src/commands/create.js

```diff
@@ -56,7 +56,7 @@ export default asyncCommand({
 		git: {
 			description: 'Initialize version control using git',
 			type: 'boolean',
-			default: true
+			default: false
```

Comment:
I think we are trying to hard here. User know how to initilize a git repo. Just leave it to user.